src/codeu/chat/client/ClientUser.java

```diff
@@ -150,4 +151,21 @@ public String showUserInfo(String uname) {
   public static void printUser(User user) {
     System.out.println(getUserInfoString(user));
   }
+
+  // Check for user in database
+  public User checkUserInDatabase(String username, String pswd){
+    // If user matches with the username and password retreive the User from the server model if don't return null
```

Comment:
nit:
I would write: 
"If user matches with the username  and password, retrieve the user from the server model. If not, return null."